backend/config/plugins.js

```diff
@@ -1,11 +1,30 @@
 const oneYear = 86400 * 365;
+const SENTRY_DEV_DSN =
+   'https://95ab6917c0234f80b99bb0be8e720355@o186239.ingest.sentry.io/6475315';
+const SENTRY_PROD_DSN =
+   'https://003ddade86504df5aa49247ba36031e7@o186239.ingest.sentry.io/6469069';
+const isProd = process.env.NODE_ENV === 'production';
 
 module.exports = ({ env }) => {
    const exports = {
       seed: {
          enabled: true,
          resolve: './src/plugins/seed',
       },
+      sentry: {
+         enabled: true,
+         config: {
+            dsn: isProd ? SENTRY_PROD_DSN : SENTRY_DEV_DSN,
```

Comment:
Why did we choose this means over taking in an env variable for the DSN (which also allows it to be unset)?

Reply:
No good reason, just an artifact of me developing. Changed in https://github.com/iFixit/react-commerce/pull/321/commits/5a9ec5be3cacdf6740dbc79eba1c8496a969f4b0 thanks!